upgrade_instructions.md

```diff
@@ -0,0 +1,18 @@
+# Update Learn CLI
+Depending on how you installed your cli tool, please use the apropiate method to upgrade below:
+
+## Homebrew
+Homebrew: `brew upgrade learn`
```

Comment:
```suggestion
Homebrew: `brew tap gSchool/learn && brew upgrade learn`
```

I believe some users have to tap again to update to the latest, otherwise it just re-installs their latest downloaded version, not necessarily the latest version.